lib/path.js

```diff
@@ -186,6 +186,7 @@ const win32 = {
     let resolvedDevice = '';
     let resolvedTail = '';
     let resolvedAbsolute = false;
+    let slashCheck = false;
```

Comment:
I believe it is possible to simplify the code by changing this to a string that we always append to the output and that is either an empty string or the slash.